<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PortalWindow</class>
 <widget class="QWidget" name="PortalWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>传送门修改</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>13</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>行</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>13</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>列</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="BlackRow_1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>6</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="BlackColumn_1">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>9</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>50</y>
     <width>13</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>行</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="WhiteRow_1">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>80</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>6</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="WhiteColumn_1">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>80</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>9</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>50</y>
     <width>13</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>列</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>82</y>
     <width>19</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>黑1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>122</y>
     <width>21</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>黑2</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="BlackRow_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>6</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="BlackColumn_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>120</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>9</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>82</y>
     <width>19</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>白1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>122</y>
     <width>21</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>白2</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="WhiteRow_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>120</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>6</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="WhiteColumn_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>120</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>9</number>
   </property>
  </widget>
  <widget class="QPushButton" name="SetBlackPortal">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>150</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SetWhitePortal">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>150</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="ActivePortal">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>90</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>启动传送门</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="LockPortal">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>20</y>
     <width>90</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>固定传送门</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
